src/tools/CMakeLists.txt

```diff
@@ -66,6 +66,12 @@ if(AXOM_ENABLE_QUEST)
     install(TARGETS              mesh_tester
             DESTINATION          bin
             )
+
+    # Add separable compilation flag
+    if (ENABLE_HIP)
+        blt_add_target_compile_flags(TO mesh_tester FLAGS "-fgpu-rdc")
+        blt_add_target_link_flags(TO mesh_tester FLAGS "-fgpu-rdc")
+    endif()
```

Comment:
+1 (I think)

Does this flag need to be use more generally for examples and tools?
If so, should we put it in a flags variable (possibly in BLT)?

Reply:
This flag is the hip-equivalent of CUDA_SEPARABLE_COMPILATION CMake flag.
I tried compiling axom with cuda or hip _and without the separable compilation flag_, and found `mesh_tester` to be the only target that required the flags in both cases. So for now, the flag is only needed here.

BLT has an open issue (LLNL/blt#367) tracking this feature request.